plugins/osdn/common.go

```diff
@@ -10,33 +10,22 @@ import (
 	log "github.com/golang/glog"
 
 	"github.com/openshift/openshift-sdn/pkg/netutils"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
+
+	osclient "github.com/openshift/origin/pkg/client"
 	osapi "github.com/openshift/origin/pkg/sdn/api"
 
 	kapi "k8s.io/kubernetes/pkg/api"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/container"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/container"
 	kerrors "k8s.io/kubernetes/pkg/util/errors"
 	kexec "k8s.io/kubernetes/pkg/util/exec"
 	kubeutilnet "k8s.io/kubernetes/pkg/util/net"
 )
 
-type PluginHooks interface {
-	PluginStartMaster(clusterNetwork *net.IPNet, hostSubnetLength uint) error
-	PluginStartNode(mtu uint) error
-
-	SetupSDN(localSubnetCIDR, clusterNetworkCIDR, serviceNetworkCIDR string, mtu uint) (bool, error)
-
-	AddHostSubnetRules(subnet *osapi.HostSubnet) error
-	DeleteHostSubnetRules(subnet *osapi.HostSubnet) error
-
-	AddServiceRules(service *kapi.Service, netID uint) error
-	DeleteServiceRules(service *kapi.Service) error
-
-	UpdatePod(namespace string, name string, id kubetypes.DockerID) error
-}
-
 type OsdnController struct {
-	pluginHooks        PluginHooks
 	pluginName         string
+	multitenant        bool
```

Comment:
We don't need `multitenant` anymore?
Now we got the pluginName, can use IsOpenShiftMultitenantNetworkPlugin(oc.pluginName) where ever required.